### PR TITLE
fix(Communities): Fix membership requests popup

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #6799

### What does the PR do

Fix listview size

### Affected areas

MembershipRequestsPopup

### Screenshot of functionality (including design for comparison)

- [ ] I've checked the design and this PR matches it

Снимок экрана 2022-08-05 в 17 20 28

### CODE REVIEW

File: ui/app/AppLayouts/Chat/popups/community/MembershipRequestsPopup.qml [hunk 1]
```diff
@@ -51,7 +51,8 @@ StatusModal {
 
             StatusListView {
                 id: membershipRequestList
-                anchors.fill: parent
+                width: parent.width
```
[the remark this answers is not in the document]
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/popups/community/MembershipRequestsPopup.qml:54]: it doesn't work :) do you know why? cos of `This property holds the width available to the contentItem after deducting horizontal padding from the width of the control` base on `contentItem` sizes, which is `zero` if your content item zero sized by default :)

[the remark this answers is not in the document]
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/popups/community/MembershipRequestsPopup.qml:54]: @igor-sirotin

[the remark this answers is not in the document]
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/popups/community/MembershipRequestsPopup.qml:54]: this one? :)

[the remark this answers is not in the document]
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/popups/community/MembershipRequestsPopup.qml:54]:
```
    readonly property int availableWidth: width - leftMargin - rightMargin
    readonly property int availableHeight: height - topMargin - bottomMargin
``` which base on width and height? :)

[the remark this answers is not in the document]
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/popups/community/MembershipRequestsPopup.qml:54]: looks like you asking about `width: width - leftMargin - rightMargin`

[the remark this answers is not in the document]
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/popups/community/MembershipRequestsPopup.qml:54]: Anyway its a wrong implementation of `StatusListView` cos in your case it should use implicit sizes of delegates. But those properties suitable just for internal usage.

**Review by MishkaRogachev — APPROVED** [review]

**Review by PascalPrecht — APPROVED** [review]

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**